cloud/packet/packetSDK/sdk.go

```diff
@@ -0,0 +1,43 @@
+// Copyright © 2017 The Kubicorn Authors
```

Comment:
We would appreciate tests here. [Example of Happy/Sad tests can be found here](https://github.com/kris-nova/kubicorn/blob/master/cloud/digitalocean/godoSdk/sdk_test.go).

Reply:
Sure thing. Will add.